pkg/generated/clientset/versioned/typed/climanager/v1/climanager.go

```diff
@@ -31,6 +32,8 @@ type CliManagerInterface interface {
 	List(ctx context.Context, opts metav1.ListOptions) (*v1.CliManagerList, error)
 	Watch(ctx context.Context, opts metav1.ListOptions) (watch.Interface, error)
 	Patch(ctx context.Context, name string, pt types.PatchType, data []byte, opts metav1.PatchOptions, subresources ...string) (result *v1.CliManager, err error)
+	Apply(ctx context.Context, name string, obj *v1.CliManager, options metav1.ApplyOptions, subresources ...string) (*v1.CliManager, error)
```

Comment:
Checking the upstream [core/v1](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/kubernetes/typed/core/v1/pod.go#L67-L73) the newNNN function is defined differently with `gentype.NewClientWithListAndApply`. Making the generated code shorter. Worth checking whether the same can be done here.

Reply:
After regenerating codes, code has switched to use `gentype.NewClientWithListAndApply`.